### ISSUE TITLE
Marker detection + Positioning

### ISSUE THREAD

**Issue by sexybiggetje** [jeromeetienne/AR.js#162]
Hi,

In order to generate some kind of "Holodeck" effect I would like to use three.js for rendering a scene, and basically use a cubicle with 4/6 markers to do some form of positioning within the cubicle. This raises some questions:

1) How would I approach this positioning part?
2) I see on "Projects" tab that you had issues with multimarker. Does that impose a problem?
3) In the videos and examples the objects are shaky, that would couse nausea in a mixed/virtual reality setup ofcourse.

So basically I want to walk around in a VR scene. I think this is something you mentioned on twitter as well?

**Comment by jeromeetienne** [comment]
you can walk around in VR using AR.js when running on tango devices. 

Source is available here -https://github.com/jeromeetienne/AR.js/blob/master/aframe/examples/vr-camera-controls.html

**Comment by sexybiggetje** [comment]
Ah yes, and what about non-tango devices, would that be possible? Because 1) I don't have a tango device yet and 2) others don't :-).

**Comment by jeromeetienne** [comment]
currently slam is unfortunatly not supported in pure-webtech.

So a tango device is currently required. When we support ARKit, you will be able to use a iphone for the same thing.

i hope it helps

**Comment by sexybiggetje** [comment]
Despite hating iOS that might be a feasable option (big userbase). Thanks for your effort and anwsers.
For now the tech isn't ready for what I want. <3 might think of something else that is usable with your current tech.
